src/web-rtc-client.js

```diff
@@ -620,12 +620,12 @@ export default class WebRTCClient extends Emitter {
 
   getAudioDeviceId(): ?string {
     // $FlowFixMe
-    return typeof this.audio === 'object' && 'deviceId' in this.audio ? this.audio.deviceId.exact : null;
+    return this.audio && typeof this.audio === 'object' && 'deviceId' in this.audio ? this.audio.deviceId.exact : null;
   }
 
   getVideoDeviceId(): ?string {
     // $FlowFixMe
-    return typeof this.video === 'object' && 'deviceId' in this.video ? this.video.deviceId.exact : null;
+    return this.audio && typeof this.video === 'object' && 'deviceId' in this.video ? this.video.deviceId.exact : null;
```

Comment:
something tells me you might want to test for `this.video` instead :)

Reply:
good catch